settings.gradle

```diff
@@ -20,4 +20,7 @@ plugins {
 rootProject.name = 'reactor'
 
 
-include 'reactor-core', 'reactor-test'
\ No newline at end of file
+include 'reactor-core', 'reactor-test'
+
+// See https://docs.gradle.org/6.5.1/release-notes.html. Drop once using Gradle 7.0+
```

Comment:
I don't think we need that given that we never have pre-GA dependencies

Reply:
My rationale is, given this *will* be the default (in 7+), enable it and see potential issues sooner than later.